### PR TITLE
Loader update

### PR DESCRIPTION
**Pull request by guybedford**
This includes support for:

* esm package (https://github.com/zeit/ncc/issues/361, implemented in https://github.com/zeit/webpack-asset-relocator-loader/pull/27)
* Reduce unnecessary asset emissions outside project boundary (https://github.com/zeit/ncc/issues/359, implemented in https://github.com/zeit/webpack-asset-relocator-loader/pull/22)
* Express templates engine support (https://github.com/zeit/ncc/issues/364, implemented in https://github.com/zeit/webpack-asset-relocator-loader/pull/25)
* Fix Webpack not found bug for branched requires (https://github.com/zeit/ncc/issues/358, implemented in https://github.com/zeit/webpack-asset-relocator-loader/pull/24)

### CODE REVIEW

**Comment by guybedford** [comment]
Seems there was a regression, looking into the fix.

**Comment by styfle** [comment]
@guybedford Please add integration tests for `esm`, `consolidate`, `fluent-ffmpeg`, and if possible the TS dist bug.

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "@google-cloud/firestore": "^0.19.0",
     "@sentry/node": "^4.3.0",
     "@tensorflow/tfjs-node": "^0.3.0",
-    "@zeit/webpack-asset-relocator-loader": "0.4.1",
+    "@zeit/webpack-asset-relocator-loader": "0.4.2",
```
Comment (styfle) [inline package.json]: Please add integration tests for esm, consolidate, fluent-ffmpeg, and if possible the TS dist bug.

**Comment by styfle** [comment]
Does this PR have https://github.com/zeit/webpack-asset-relocator-loader/pull/29

File: package.json (hunk 1 above)
Comment (guybedford) [reply package.json]: Sure, I've added the integration tests.

**Comment by guybedford** [comment]
It does now.

**Pull request merged**